msgcntr/messageforums-app/src/webapp/jsp/discussionForum/statistics/dfStatisticsListByTopic.jsp

```diff
@@ -155,6 +157,17 @@
 				var menuLinkSpan = menuLink.closest('span');
 				menuLinkSpan.addClass('current');
 				menuLinkSpan.html(menuLink.text());
+				var sakaiReminder = new SakaiReminder();
```

Comment:
Try to use const wherever you can. If you *really* have to reassign variables, use let. const makes js code a load more reliable. You know it can't be reassigned anywhere, which is really nice.

https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/